app/main.py

```diff
@@ -1,8 +1,26 @@
 class Person:
-    # write your code here
-    pass
+
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+
+        Person.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    person_list = []
+    for one_person in people:
+        person_list.append(
+            Person(name=one_person["name"], age=one_person["age"])
+        )
```

Comment:
When we have only append operation and some if statements, then we are free to use list comprehension :)